astro/src/content/docs/get-started/download-and-install/docker.mdx

```diff
@@ -222,15 +127,25 @@ When running in development runtime mode, silent mode was enabled based upon the
 
 The Docker Compose file references the `latest` tag, but that tag is not dynamic. It is only the latest at a point in time. To get the most recently released image, you have a couple of options.
 
-* Edit the Docker Compose file and specify a specific version. This is a good idea for a production deployment.
-* Remove the old image first: `docker rmi <old image id>`. This command may prompt you to remove containers using that image. Since all state is stored in the database, you can safely remove the containers. Then, when you restart Docker Compose, the latest image will be retrieved.
-* Pull the latest image with this command: `docker pull fusionauth/fusionauth-app:latest`.
+* Pull the latest image with this command: `docker pull fusionauth/fusionauth-app:latest` or `docker compose pull` and recreate your deployment with `docker compose up -d` which will recreate every container where a new image is available.
+* Edit the Docker Compose file and specify a specific version. This is a good idea for a production deployment. With `docker compose pull` you can only pull the specified image, or just run `docker compose up -d` which will pull the image and recreate the container at the same time.
+* Remove all images with `docker rmi <IMAGE ID>` selected from the list of IMAGE IDs which you can show with `docker images`. This requires that the image isn't used and therefore may prompt you to remove containers using it. Since all state of FusionAuth is stored in the database, and for docker you use volumes to persist data, you can safely remove the containers. After that you can use `docker compose pull`, `docker compose build` and `docker compose up -d` accordingly to get images specified the in the `docker-compose.yml`.
```

Comment:
I've never needed to use `docker compose build`. Does `docker compose up -d` do that implicitly?

Reply:
There are some scenarios where you use `docker compose build` especially in combination with `--pull` and `--no-cache` in scenarios where you use the latest tag. Or when you want to first test the build on its own after an version update.

Reply:
Okay, that's fine, let's leave it in there then.